.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+> If you want to report a bug or request assistance please use the following template.
+
+---
+
+### Bug Report / Assistance Request
+
+#### Environment
+
+* DART version: 6.3.0 / master
```

Comment:
Maybe we should add the word `Select ` in front of each of these lines in order to convey that we want the user to specify which one they are using. We should also probably end each line with ` / other (please specify)`